src/librustdoc/html/highlight/fixtures/sample.rs

```diff
@@ -0,0 +1,9 @@
+#![crate_type = "lib"]
+
+#[cfg(target_os = "linux")]
+fn main() {
+    let foo = true && false || true;
```

Comment:
Can you add cases for

```
let _ = &foo;
let _ = &&foo;
let _ = *foo;
```

These were special cased in the previous code.

Reply:
Urgh, forgot to submit comment. Test added

Before (actual rustdoc output)

![image](https://user-images.githubusercontent.com/1711539/91470131-9cb2f880-e894-11ea-9ff7-fced8e491735.png)

After (unittest output) (I don't see any differences)

![image](https://user-images.githubusercontent.com/1711539/91470143-a0df1600-e894-11ea-8d8c-02c2a95ff313.png)

Both versions miscolorize the `&&`.

Reply:
Yep, I'm aware of the `&&` issue, just want to make sure we have some data for when it's fixed :)

Reply:
Yeah... It's nice that we can now easily add all the old special-cases to the tests, expect-test make this super-nice :D